topics/variant-analysis/tutorials/MTB_transmission/tutorial.md

```diff
@@ -0,0 +1,812 @@
+- Clustering does not provide information about particular transmission events nor their directionality (who infected whom).
+- Clustering is very much influenced by sampling. Lower sampling proportions and shorter sampling timeframes lead to lower clustering rates that shoud not be confounded with lack of transmission.
+contributors:
+- GaloGS
+- dbrites
+- cstritt
+---
+
+# Pre-requisites
+This tutorial belongs to *Day 3: Evolutionary epidemiology: using phylogenetics to understand DR emergence and Mtb transmission* of the workshop [Mycobacterium tuberculosis NGS made easy: data analysis step-by-step](https://gallantries.github.io/video-library/events/mtb-ngs/program.html). It assumes you went through the materials
+of *Day 2*, and you have watched the respective webinars and completed the tutorial on [MTB variant analysis](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html).
+
+# Introduction
+{:.no_toc}
+
+
+Now you are familiar with the process of genome sequencing, quality control of sequencing data, mapping and
+variant calling. Additionally, you know how to run `TB-profiler` in order to detect drug resistance-conferring
+mutations and predict drug-resistant phenotypes. However you have only analyzed one sample unrelated to our study for the purpose of
+learning. Before starting the transmission analysis, we would need to analyze the 20 samples that we
+have been asked to analyze. The process to follow is exactly the same for any sample, meaning that
+we would need to analyze the 20 samples following the same steps... twenty times??? Of course not.
+
+The way you can perform the same analysis tasks on multiple samples at a a time in Galaxy (being it
+a simple step like running Trimmomatic, or a complete analysis pipeline composed of many different steps)
+is by using `Dataset collections`. To know more about dataset collections,
+you can [have a look at the following material](https://galaxyproject.org/tutorials/collections/), however the idea is very simple:
+all samples to be analyzed are merged into a "dataset collection", and then you analyze that "collection"
+as you would analyze a single sample, with the difference that all steps will be performed for each
+sample individually. Do not worry if you do not fully understand this right now, we will be creating
+a dataset collection at the beginning of this tutorial and we will analyze it.
+
+To save you some time, (and server load) we have run the pipeline that you used in the previous tutorial for
+the 20 samples that we want to analyze. Thus, we now have 20 VCF files that describe the
+mutations found for each of the samples. **These 20 VCFs files will be the starting point of this tutorial.**
+ If you want to perform the mapping and variant calling for
+all of the samples, feel free to do it. You can find the respective [FASTQ files here](https://zenodo.org/record/5911437), and
```

Comment:
This should be in the zenodo_link at top I think

Reply:
This is the only thing I am not sure about. For the workshop/tutorial we have created two repositories. One only has the fastq files (because this repository will be also shared and used for other purposes). The other one has the analysis "results", so only the VCFs and the output of TB-profiler. The zenodo link at top points to the second one, while in this paragraph we point to a second, optional, Zeonodo repository.